### DISCUSSION TITLE
Kompatible Netzwerk Stromzähler

### DISCUSSION THREAD

**Discussion by srype** [camueller/SmartApplianceEnabler#226] [Installation und Betrieb]
Welche Ethernet Stromzähler sind mit dem SAE kompatibel? Funktioniert das mit der ET Reihe von Victron?

**Comment by camueller** [comment]
Ethernet ist ja nur die Transport-Ebene. Darüber kann man Modbus/TCP oder HTTP machen. Wenn der Stromzähler eines von beiden kann, sollte es gehen.

**Reply by srype** [reply]
Wie binde ich dann Zähler über das Netzwerk ein? Gibt es eine Liste für kompatible Geräte? In meinem Fall funktionieren S0 oder RS485 nicht, da die Zähler weit vom Raspi entfernt sind und die Übertragung teilw. per Fiber läuft.

**Reply by premultiply** [reply]
Einfacher RS485-IP-Umsetzer. Sowas wie Protoss-PE11-H: https://de.aliexpress.com/item/4001085404767.html?spm=a2g0o.seodetail.topbuy.1.5345489aOamdfm
Die können sogar noch optional das Protokoll von ModbusRTU<->ModbusTCP umsetzen bei Bedarf.

**Reply by srype** [reply]
Lässt sich das noch einfacher umsetzen? Den SAE will ich auf dem Weingut meines Onkels montieren. Dort sollte im Sommer eine Kühlanlagen mit 32A/3~ Anschlussleistung gemessen und gesteuert werden. Dazu noch mehrere mobile Verbraucher wie Heizgebläse für den Winter etc.
Die Anbindung ins Netzwerk sollte daher per Wifi laufen. Plan war, dass ich mir dafür ein paar Gehäuse mit Stromzähler, Schütz und dazugehöriger Netzwerktechnik baue. Sollte man dann einfach einstecken können und fertig. Ausserdem habe ich nicht immer direkt Zugriff da ich in 100km Entfernung wohne. Ist das mit dem SAE überhaupt so eine gute Idee?
